.github/workflows/release.yml

```diff
@@ -0,0 +1,103 @@
+name: ci
+
+# This worflow needs this secret:
+#
+# DOCKERPASSWORD = Docker Hub token
+
+on:
+  push:
+    branches: [master]
+    tags: 'v*.*.*'
+  schedule:
+    - cron: "0 13 * * 1"
+
+env:
+  PLATFORMS: "linux/amd64,linux/arm/v7,linux/arm/v6,linux/arm64, linux/ppc64le"     # Build for which platforms
+  IMAGENAME: "ansemjo/speedtest"                                     # Name of the image
+  DEFAULT_TAG: "latest"                                              # Which tag is beeing used if we are building for master/main branch 
+  DOCKER_USER: "ansemjo"                                             # Which user to use to login to DockerHub
```

Comment:
Would it make sense to also parametrize the user here as a "secret" and also substitute it in the `IMAGENAME` like `${{ secrets.DOCKER_USER }}/speedtest`? That way somebody who forks this repository would not need to change the workflow at all, only set the correct username?

Reply:
Could make sense. At least for us two because I needed to change it after my last sucessfull run ;-)

Reply:
thatwouldbegreat.jpeg :P

If you'd make both `DOCKER_USER` and `DOCKER_TOKEN` secrets and parametrize the image name as described above, I'd be happy to merge this PR. :)

Reply:
I named it DOCKERHUB_* not DOCKER_* just to make sure for which registry it is (makes it easier to add more credentials for other registries as well)